storage/tables/index.md

```diff
@@ -141,7 +141,7 @@ and you import the following data to the table:
 
 |name|age|money|
 |---|---|---|
-|Annie|30|$500|
+|Annie|30|$500000|

```

Comment:
zvyšoval jsi to z nějakýho konkrétního důvodu nebo jen aby to vypadalo jako reálnější platy? pokud takhle, tak potom by update Johna neměl bejt $340 ale třeba $350000

Reply:
zvysoval jsem to prave pro to, aby bylo jasneji videt, ktere radky jsou z prvni a ktere z druhe tabulky